### PR TITLE
Morph this theme to run on Fork 3.7 and up

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by WouterSioen** [comment]
I don't know who manages this, but can you create a tag before merging this?

**Comment by ghost** [comment]
Maybe @Diewy manages this?

**Comment by WouterSioen** [comment]
@Diewy does not work @ wijs anymore.  I think @Dextro is the only one with access to these wijs repo's?

**Comment by WouterSioen** [comment]
poke @Dextro

File: info.xml [hunk 1]
```diff
@@ -19,14 +19,14 @@
...
-		<template label="Default" path="core/layout/templates/default.tpl">
+		<template label="Default" path="Core/Layout/Templates/Default.tpl">
 			<positions>
 				<position name="main" />
 				<position name="right" />
 				<position name="doormatLeft" />
 				<position name="doormatMiddle">
 					<defaults>
-						<widget module="blog" action="recent_comments" />
+						<widget module="Blog" action="Recent_comments" />
```
Comment (jessedobbelaere) [inline info.xml]: Are all actions not CamelCased? Maybe remove the underscore?

File: info.xml [hunk 2]
```diff
@@ -48,7 +48,7 @@
...
```
Comment (jessedobbelaere) [inline info.xml]: Same here :wink:

**Comment by jessedobbelaere** [comment]
@guytarded  Added a few remarks. Also, this upgrade is not yet complete. The templates contain a lot of references to the old (not camelcased) folders. 

Focus on these strings (use them to search with your code editor): :wink: Ofcourse, that's probably not all of them, but you get an idea what to look out for

```
core/layout/templates
{$THEME_URL}/core/js
{include:core/layout/templates/head.tpl
{include:core/layout/templates/footer.tpl
{include:core/layout/templates/pagination.tpl
{$THEME_URL}/core/layout/images
```

**Comment by ghost** [comment]
Darn, looks like i've forgotten a few things.

**Pull request merged**